src/main/java/in/org/projecteka/hiu/consent/ExpiredConsentTask.java

```diff
@@ -0,0 +1,24 @@
+package in.org.projecteka.hiu.consent;
+
+import in.org.projecteka.hiu.consent.model.ConsentArtefactReference;
+import in.org.projecteka.hiu.consent.model.ConsentStatus;
+import reactor.core.publisher.Mono;
+
+import java.time.LocalDateTime;
+
+public class ExpiredConsentTask implements ConsentTask {
+    private ConsentRepository consentRepository;
+    private DataFlowDeletePublisher dataFlowDeletePublisher;
+
+    public ExpiredConsentTask(ConsentRepository consentRepository, DataFlowDeletePublisher dataFlowDeletePublisher) {
+        this.consentRepository = consentRepository;
+        this.dataFlowDeletePublisher = dataFlowDeletePublisher;
+    }
```

Comment:
Why no `@AllArgsConstructor`?

Reply:
we can. but I think constructor is also ok.

Reply:
Agree. But since we are using Lombok so I thought it would be appropriate.